### PR TITLE
[Term Entry] PyTorch Tensor Operations: .addcmul()

### LINKED ISSUE #6314: [Term Entry] PyTorch Tensor Operations: .addcmul()
### Reason/inspiration (optional)

We would like a new entry on the `.addcmul()` term under PyTorch. The entry should go in a new file under `docs/content/pytorch/concepts/tensor-operations/terms/addcmul/addcmul.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept
- An `Example` section that provides an example demonstrating the concept in use

Please refer to the [term entry template](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [x] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by mamtawardhani**
### Description
Added [Term Entry] PyTorch Tensor Operations: .addcmul()

### Issue Solved
Closes #6314 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/addcmul/addcmul.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+Tags:
+  - 'Deep Learning'
+  - 'PyTorch'
+  - 'Tensors'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/machine-learning'
+---
+
+**`.addcmul()`** is a PyTorch [tensor](<(https://www.codecademy.com/resources/docs/pytorch/tensors)>) operation that performs element-wise multiplication of two tensors and then adds a third tensor to the result, optionally scaled by a value. This operation combines multiplication and addition in a single efficient function call.
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/addcmul/addcmul.md:16]:
```suggestion
**`.addcmul()`** is a PyTorch [tensor]((https://www.codecademy.com/resources/docs/pytorch/tensors)) operation that performs element-wise multiplication of two tensors and then adds a third tensor to the result, optionally scaled by a value. This operation combines multiplication and addition in a single efficient function call.
```

File: content/pytorch/concepts/tensor-operations/terms/addcmul/addcmul.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+z: tensor([1.5, 2.5, 3.5])
+
+Result of torch.addcmul(x, y, z, value=2.0): tensor([1.3000, 3.0000, 5.1000])
+Original x is unchanged: tensor([1., 2., 3.])
+
+After x.addcmul_(y, z, value=2.0), x is now: tensor([1.3000, 3.0000, 5.1000])
+x now equals the previous result: True
+```
+
+To explore more, check out this [Intro to PyTorch and Neural Networks](https://www.codecademy.com/enrolled/courses/intro-to-py-torch-and-neural-networks) course, where you’ll learn to create, train, and test neural networks.
```
Comment (Radhika-okhade) [inline content/pytorch/concepts/tensor-operations/terms/addcmul/addcmul.md:163]:
```suggestion
To learn more about how to create, train, and test neural networks, check out this [Intro to PyTorch and Neural Networks](https://www.codecademy.com/enrolled/courses/intro-to-py-torch-and-neural-networks) course on Codecademy.
```

**Comment by mamtawardhani** [comment]
Made the changes @Radhika-okhade 🫡
